domains/whisker.is-a.dev.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "Whisker-dev",
+    "email": "ibrahimzayd.007@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://whisker-dev.github.io/Main-website/"
```

`CNAME` record must contain a domain only, remove `https://` and `/Main-website/` : 
```json
"CNAME": "whisker-dev.github.io"
```

---

wait i can suggest changes
```suggestion
    "CNAME": "whisker-dev.github.io"
```